docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:20.04
+RUN \
+  # configure the "jhipster" user
+  groupadd jhipster && \
+  useradd jhipster -s /bin/bash -m -g jhipster -G sudo && \
+  echo 'jhipster:jhipster' |chpasswd && \
+  mkdir /home/jhipster/app && \
+  export DEBIAN_FRONTEND=noninteractive && \
+  export TZ=Europe\Paris && \
```

Comment:
Why not use `UTC`?

Reply:
Doesn't have any specific reason . further it doesn't anything as it is for Inside the running Container  , as soon as generation is done , the running container will destroyed . I kept it there to avoid any issue related to Time Zone